Documentation/policy/language.rst

```diff
@@ -1056,3 +1059,36 @@ A more pod-specific solution is to configure ``ndots`` appropriately for each
 Pod, via ``dnsConfig``, so that the search list is not used for DNS lookups
 that do not need it. See the `Kubernetes documentation <https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-config>`_
 for instructions.
+
+
+
+.. _HostPolicies:
+
+Host Policies
+=============
+
+Host policies take the form of a `CiliumClusterwideNetworkPolicy` with a
+:ref:`NodeSelector` instead of an `EndpointSelector`. Host policies can have
+layer 3 and layer 4 rules on both ingress and egress. They cannot, however,
+have layer 7 rules.
+
+Host policies apply to all the nodes selected by their :ref:`NodeSelector`. In
+each selected node, they apply only to the host namespace, including
+host-networking pods. They therefore don't apply to communications between
+pods, between pods and a service, or between the outside of the cluster and any
+of the former.
```

Comment:
nit: A service could be backed by the host, no?

Also, `between the outside of the cluster and any of the former.` is very ambiguous. I think it's clearer to just spell out about traffic between non-hostnetwork pods and network locations outside of the cluster.